docs/source/pages/using.rst

```diff
@@ -314,7 +314,53 @@ Analyzing a locally cloned repository
 
 If you have a local repository that you want to analyze, Macaron also supports running the analysis against a local repository.
 
-Assume that the dir tree at the local repository has the following components:
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Analyzing a repository whose git service is not supported by Macaron
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If the repository remote URL is from an unknown git service (see :ref:`Git Services <supported_git_services>` for a list of supported git services in Macaron), Macaron won't recognize it when analyzing the repository.
+
+You would need to tell Macaron about that git service through the ``defaults.ini`` config.
+For example, let's say you want to analyze the Bitbucket repository at ``https://bitbucket.org/snakeyaml/snakeyaml``. First, you need to create a ``defaults.ini`` file in the current workspace with the following content:
```

Comment:
I'm not sure using  `bitbucket.org` as the example is a good idea, since this is a well-known public git service that in principle we should support at some point, rendering this example meaningless (and the same goes for any other real public git service one might choose). Using a nonexistent hostname as the example (e.g. `git.example.com`) would probably be better, at the cost of making the example not concretely executable.

Reply:
I agree. Using `bitbucket.org` is not the best way to demonstrate an unknown generic git service.